astropy/io/ascii/tests/test_write.py

```diff
@@ -673,7 +673,9 @@ def test_write_quoted_empty_field(fast_writer):
 def test_write_overwrite_ascii(format, fast_writer, tmpdir):
     """Test overwrite argument for various ASCII writers"""
     filename = tmpdir.join("table-tmp.dat").strpath
-    open(filename, 'w').close()
+    with open(filename, 'w'):
+        # create empty file
```

Comment:
Maybe do this?
```
Path(filename).touch()
```
https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch
https://github.com/python/cpython/blob/4ff38870b1de8a3add5357edf125c2866bc42b54/Lib/pathlib.py#L1218

Reply:
@saimn , are you planning to address this or are you leaving it out for PY2 compat?

Reply:
@pllim - I prefer the backport option, so keeping the current version.